### PR TITLE
migrate players

### PR DESCRIPTION
**Pull request by elliansjm**

### CODE REVIEW

File: src/warzone/src/warzone/state/Startup.java [hunk 1]
```diff
@@ -9,9 +9,12 @@
  *  that the command is invalid. 
  */
 public class Startup extends GamePlay {
+	
+	private StartupService d_startupService;
```
Comment (amyhcy) [inline src/warzone/src/warzone/state/Startup.java:24]: javadoc missing

File: src/warzone/src/warzone/state/Startup.java [hunk 2]
```diff
@@ -21,30 +24,79 @@ public void next() {
...
+	public void populatecountries(){
+		boolean result = d_startupService.assignCountries();
+		if(result == false) {
+			GenericView.printError("Must have more than 2 players, and map have at least the same number of countries as players ");
+		}
+		else {
+			GenericView.printSuccess("Succeed to assign all the countries to players");
+		}
+	}
+
```
Comment (amyhcy) [inline src/warzone/src/warzone/state/Startup.java]: javadoc missing

File: src/warzone/src/warzone/state/Startup.java [hunk 3]
```diff
@@ -21,30 +24,79 @@ public void next() {
...
-	
-	 public void populatecountries(){
-		 //todo;
-	 }	
-	 
-	 public void reinforcement(){
-		 printInvalidCommandMessage();
-	 }	
- 
 	public void issueOrder(){
```
Comment (amyhcy) [inline src/warzone/src/warzone/state/Startup.java:110]: javadoc missing

File: src/warzone/src/warzone/state/Startup.java [hunk 4]
```diff
@@ -21,30 +24,79 @@ public void next() {
...
-		 printInvalidCommandMessage();
-	 }	
+		printInvalidCommandMessage();
+	}
+
```
Comment (amyhcy) [inline src/warzone/src/warzone/state/Startup.java:113]: javadoc missing

**Review by amyhcy — COMMENTED** [review]
missing some javadoc

**Review by fzuray — APPROVED** [review]

**Pull request merged**